deps/ncrypto/ncrypto.h

```diff
@@ -292,10 +293,33 @@ class Cipher final {
                              const CipherParams& params,
                              const Buffer<const void> in);
 
+  static bool IsValidGCMTagLength(unsigned int tag_len);
+
  private:
   const EVP_CIPHER* cipher_ = nullptr;
 };
 
+// ============================================================================
+// DSA
+
+class Dsa final {
+ public:
+  Dsa();
```

Comment:
```suggestion
  Dsa() = default;
```

Reply:
I generally prefer the explicit constructor but we can revisit this after once ncrypto is in a standalone repo